mmv1/products/managedkafka/Cluster.yaml

```diff
@@ -154,9 +154,9 @@ properties:
     description: "Capacity configuration at a per-broker level within the Kafka cluster. The config will be appled to each broker in the cluster."
     ignore_read: true
     properties:
-      - name: 'diskSizeGb'
+      - name: 'diskSizeGib'
```

Comment:
It's possible to avoid the breaking change entirely by using `api_name` ([reference](https://googlecloudplatform.github.io/magic-modules/reference/field/#api_name) which remaps a Terraform field name to a different API name.

Another option could be to leave the field as GB and use a customer flattener and expander to convert that to/from GIB when talking to the API.

OTOH, there is a difference between GiB and GB. If your logs show that no users have been using this field it _might_ be OK (is the service account that the Terraform integration tests use allow listed for this field? The nightly test runs are passing with the field).

When will the old version of the field be turned down? As you have probably gathered, we take breaking changes pretty seriously :-)

Reply:
I messaged you offline with a better explanation regarding some internal details, but essentially neither the old GB field, nor the new GiB field are externally visible for users, so I imagine that there are nobody using the GB field we are replacing

> When will the old version of the field be turned down?

The field is already unused in our code, but we can't delete it yet (for example, we reference this field here in Terraform)